src/anomalib/data/utils/augmenter.py

```diff
@@ -158,7 +158,8 @@ def augment_batch(self, batch: Tensor) -> tuple[Tensor, Tensor]:
 
         # Apply perturbations batch wise
         if isinstance(self.beta, float):
-            beta = self.beta
+            beta = torch.rand(batch_size) * self.beta
+            beta = beta.view(batch_size, 1, 1, 1).expand_as(batch).to(batch.device)  # type: ignore
```

Comment:
I'm not sure about this one. The intended use of passing a single value is to have a fixed beta value across images, which might be preferred in some cases. For random beta values, the user could pass a tuple to set the range of possible beta values. This is already the default behaviour of DRAEM, so I don't think we need to change the augmenter for this.

The only difference that I can see is that the paper mentions a range of [0.1, 1.0], while the default range of the augmenter is [0.2, 1.0]. We could easily address this by either hard-coding the correct range in the lightning model and passing it to the constructor of the augmenter, or by making it configurable from the `model` section of the config file.

Reply:
This makes sense yes. I changed this based on [original implementation](https://github.com/VitjanZ/DRAEM/blob/main/data_loader.py#L126), but as you said this can be covered by simply setting the range to [0, 0.8].
I will revert this, while I think that it would be good idea to make it part of config.

Reply:
This is now reverted.

Reply:
> I think that it would be good idea to make it part of config.

I agree. Would be good if you could add it to this PR. Just be aware that there is a small discrepancy between the paper and the original implementation. The paper defines beta as the weight of the anomalous perturbations and proposes a beta range of [0.1, 1.0]. The implementation on the other hand, samples beta from the [0, 0.8] range and then uses (1-beta) as the weight of the perturbations, which effectively results in a [0.2, 1.0] range (which is the same range that we were using in Anomalib).

I would propose to use the numbers mentioned in the paper for the default configuration

Reply:
I now added this to config.